Apps/Laboratory/test/unit/Services/LaboratoryServiceTests.cs

```diff
@@ -188,11 +207,16 @@ public async Task ShouldGetLaboratoryOrders(ResultType expectedResultType, bool
                 ResultStatus = expectedResultType,
                 PageSize = 100,
                 PageIndex = 1,
-                ResourcePayload = new() { Result = laboratorySummary },
+                ResourcePayload = new()
+                {
+                    Result = laboratorySummary, LoadState = new PhsaLoadState
+                        { RefreshInProgress = refreshInProgress },
```

Comment:
Nit: code formatting is off.

```suggestion
                    Result = laboratorySummary,
                    LoadState = new PhsaLoadState { RefreshInProgress = refreshInProgress },
```